apps/web/lib/webhooks/constants.ts

```diff
@@ -16,3 +18,16 @@ export const WEBHOOK_TRIGGER_EVENTS = [
   ...WEBHOOK_TRIGGER_EVENTS_GROUPED_BY_APP.core,
   ...WEBHOOK_TRIGGER_EVENTS_GROUPED_BY_APP.routing_forms,
 ] as const;
+
+export const WEBHOOK_TRIGGER_EVENTS_GROUPED_BY_APP_V2 = () => {
+  const { t } = useLocale();
+  return {
+    core: [
+      { value: WebhookTriggerEvents.BOOKING_CANCELLED, label: t("booking_cancelled") },
+      { value: WebhookTriggerEvents.BOOKING_CREATED, label: t("booking_created") },
+      { value: WebhookTriggerEvents.BOOKING_RESCHEDULED, label: t("booking_rescheduled") },
+      { value: WebhookTriggerEvents.MEETING_ENDED, label: t("meeting_ended") },
+    ],
+    routing_forms: [{ value: WebhookTriggerEvents.FORM_SUBMITTED, label: t("form_submitted") }],
+  };
+};
```

Comment:
Since we changing the trigger event selection to a select component we need to add labels